static/app/utils/performance/suspectSpans/suspectSpansQuery.tsx

```diff
@@ -26,7 +28,7 @@ type Props = RequestProps & {
 };
 
 function getSuspectSpanPayload(props: RequestProps) {
-  const {perSuspect, spanOps, spanGroups} = props;
+  const {perSuspect, spanOps, spanGroups, minExclusiveTime, maxExclusiveTime} = props;
```

Comment:
I'm not super familiar with how we do this elsewhere but do we normally keep the order alphabetically as it's defined in props or does it not matter?

Reply:
we keep property names in interface declarations alphabetically ordered. i'm not aware of any rules around destructuring 🤔 functionally, it shouldn't make any difference though